docs/requirements.txt

```diff
@@ -8,4 +8,5 @@ sphinx-autodoc-typehints
 myst-parser
 numpydoc
 docutils
+sphinx_rtd_theme
```

Comment:
why? we use book theme, right?

Reply:
Documentation build was giving errors, that is why I included this? Are you sure this is not a related package we need?

Reply:
I'll double check. It was removed with a reason.

Reply:
I'm resolving this for now, since this is related to docs in general. We can do a quick fix later if needed.